emulators/dosbox-x/Portfile

```diff
@@ -55,6 +55,12 @@ autoreconf.args     -fvi
 
 configure.args-append --enable-sdl2
 
+# add a slirp variant for TCP/IP emulation support
+
+variant slirp description {Use the libslirp library} {
+    depends_lib-append port:libslirp
```

Comment:
Does dosbox-x link with a library installed by the libslirp port? If so, the port must ensure that that does not happen if this variant is not selected.

Reply:
Hi @ryandesign,

I have updated my commit to disable slirp support in the default variant and to enable it only when the slirp variant is selected.

Thanks,

-ranga